roles/cdi/templates/cdi-controller-deployment.yml

```diff
@@ -15,7 +15,8 @@ spec:
       containers:
       - name: cdi-controller
         image: {{ repo_tag }}/import-controller:{{ release_tag }}
-        imagePullPolicy: Always
+#        imagePullPolicy: Always
```

Comment:
@screeley44 Scott, hi. What is this commented line for?

Reply:
sorry, left over, I can remove it